Stripe/PublicHeaders/STPPaymentMethodParams.h

```diff
@@ -23,7 +23,7 @@ NS_ASSUME_NONNULL_BEGIN
  
  @see https://stripe.com/docs/api/payment_methods/create
  */
-@interface STPPaymentMethodParams : NSObject <STPFormEncodable>
+@interface STPPaymentMethodParams : NSObject <STPFormEncodable, STPPaymentOption>
```

Comment:
is this conformance needed for this PR?

Reply:
No, this was mistakenly left in from the Standard Integration support (which will be in another PR).